addons/pharma/functions/fnc_removeItemfromMag.sqf

```diff
@@ -0,0 +1,40 @@
+#include "script_component.hpp"
+/*
+ * Author: Katalam
+ * Removes a 'bullet' from a magazine.
+ *
+ * Arguments:
+ * 0: Magazine <STRING>
+ * 1: Medic <OBJECT>
+ * 2: Patient <OBJECT>
+ * 3: Selection <STRING>
+ * 4: Classname <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ['kat_Painkiller', player, cursorTarget, 'head', 'Painkillers'] call kat_circulation_fnc_removeItemfromMag;
+ *
+ * Public: No
+ */
+
```

Comment:
Why do we need this file in the pharma addon, when it is already in the circulation mod? if you need it here, then call it like in the example